Rakefile

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'ci/reporter/rake/test_unit'
```

Comment:
Technically this is optional (since you can disable the test group). That's why it's better to use:
```ruby
begin
  require 'ci/reporter/rake/test_unit'
rescue LoadError
  # test group not enabled
else
  namespace :jenkins do
    desc nil # No description means it's not listed in rake -T
    task unit: ['ci:setup:testunit', :test]
  end
end
```